src/main/java/com/scottlogic/practice/mealscheduler/Models/Meal.java

```diff
@@ -1,25 +1,39 @@
 package com.scottlogic.practice.mealscheduler.Models;
 
-import lombok.Data;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
 import java.time.LocalDateTime;
-import java.util.UUID;
 
-@Data
+/*not using @Data class as IntelliJ says it is bad to use it with @Entity*/
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
 public class Meal {
-    private UUID Id;
+    public static Meal copyOf(Meal meal) {
+        return new Meal(meal.getId(), meal.getName(), meal.getTime(), meal.getUser());
+    }
+
+    @Id
+    @GeneratedValue
+    @Getter
+    @Setter
+    private int Id;
```

Comment:
I don't mind but is there a reason the id is an int not a uuid?

Reply:
Wanted to simplify, to make sure it works in the db (types are way simpler to map). Once the db fully works, we can bring UUID back again, if we think they are better. For me they do look a bit as an overkill.

Reply:
The @GeneratedValue may not be easily usable with UUID, let's see in the future